Modules/Office365DSC/DSCResources/MSFT_O365AdminAuditLogConfig/MSFT_O365AdminAuditLogConfig.psm1

```diff
@@ -49,16 +49,23 @@ function Get-TargetResource
             $UnifiedAuditLogIngestionEnabledReturnValue = 'Disabled'
         }
 
-        return @{
+        [void](Get-PSSession | Remove-PSSession)
+        $Result = @{
             IsSingleInstance                = $IsSingleInstance
             Ensure                          = 'Present'
             GlobalAdminAccount              = $GlobalAdminAccount
             UnifiedAuditLogIngestionEnabled = $UnifiedAuditLogIngestionEnabledReturnValue
         }
+        Write-Verbose "Returning Get-TargetResource Result"
+        return $Result
     }
     catch
     {
-        Write-Verbose 'Unable to determine Unified Audit Log Ingestion State.'
+        $ExceptionMessage = $_.Exception
+        Write-Verbose "Error Caught, exception message is: $ExceptionMessage"
+        Write-Warning 'Unable to determine Unified Audit Log Ingestion State.'
+        [void](Get-PSSession | Remove-PSSession)
```

Comment:
Since it doesn't really make sense to return a Null object that really isn't null, I would simply get rid of the null hash and return, and throw an error in here instead.

Reply:
I thought that the null return was necessary for ReverseDSC?  I can take out the ExceptionMessage, I was just using that during troubleshooting (it's how I determined that the cmdlets from the imported session weren't in scope.